diagnosisApplication/src/main/view/DiagnosisView.java

```diff
@@ -17,44 +19,76 @@ public class DiagnosisView extends JPanel {
     private final SymptomCheckerController symptomCheckerController;
     private final DiagnosisState diagnosisState;
 
+    private final JButton symptomChecker;
+
     public DiagnosisView(DiagnosisViewModel diagnosisViewModel, DiagnosisController diagnosisController,
                          SymptomCheckerController symptomCheckerController) {
         this.diagnosisViewModel = diagnosisViewModel;
         this.diagnosisController = diagnosisController;
         this.symptomCheckerController = symptomCheckerController;
         this.diagnosisState = diagnosisViewModel.getState();
 
+        int n = diagnosisState.getNumDiagnoses();
+
         JLabel title = new JLabel(DiagnosisViewModel.TITLE_LABEL);
 
         JPanel information = new JPanel();
+        for (int i = 0; i < n; i++) {
+            JPanel infoPanel = new JPanel();
+            HashMap<String, Object> currentDiagnosis;
+            if (i == 2) {
+                currentDiagnosis = diagnosisState.getDiagnosis3();
+            } else if (i == 1) {
+                currentDiagnosis = diagnosisState.getDiagnosis2();
+            } else {
+                currentDiagnosis = diagnosisState.getDiagnosis1();
+            }
+            StringBuilder builder = new StringBuilder();
+            builder.append(currentDiagnosis.get("name")).append(" (")
+                    .append(currentDiagnosis.get("accuracy")).append(" likelihood").append(")").append("\n");
+            builder.append("\t\t" + "Professional Name: ").append(currentDiagnosis.get("profName")).append("\n");
+            builder.append("\t\t" + "ICD Number: ").append(currentDiagnosis.get("icd")).append("\n");
+            builder.append("Specialists:" + "\n");
```

Comment:
Constructing the diagnosis result into user friendly text to be displayed in the view